app/src/main/res/layout/main_fragment.xml

```diff
@@ -8,78 +8,76 @@
     tools:context=".UI.MainFragment">
 
 
-        <AutoCompleteTextView
-            android:id="@+id/rcpSearch"
+        <LinearLayout
```

Comment:
Added buttons in a layout and made it relative to help it stay where it needs to be